data/dictionary/left joiner/20ab24f4-9017-4cf1-9b44-45211ec3d074.json

```diff
@@ -1,20 +1,23 @@
 {
-  "description": "Joins dictionary 'b' to dictionary 'a', by matching the values of 'a' to the keys of 'b'.",
+  "description": "Joins dictionary 'b' to dictionary `a`, by matching the values of `a` to the keys of `b`.\n\nItems from `b` with no matching value in `a` are excluded from the result.\n\nItems from `a` with no matching key in `b` are excluded from the result.\n\nExample:\n1. {\"first\":\"first_id\", \"third\":\"third_id\", \"fifth\":\"fifth_id\"}@0 received via `a`\n2. {\"first_id\":1, \"second_id\":2, \"third_id\":3}@0 received via `b`\n3. `joined` sends {\"first\": 1,\"third\": 3}@0",
   "interface": {
     "inputs": {
       "dfab5f2f-9107-481d-b7b1-19a7631381fb": {
         "type": "{string:string}",
-        "name": "a"
+        "name": "a",
+        "description": "Receives the dictionary containing the keys to join to, and the values to join by.\n\nExample:\n{\"first\":\"first_id\", \"third\":\"third_id\", \"fifth\":\"fifth_id\"}"
       },
       "be713809-1c65-4abb-9525-87ee9f75e9be": {
         "type": "{string:any}",
-        "name": "b"
+        "name": "b",
+        "description": "Receives the dictionary containing the values to join to, and the keys to join by.\n\nExample:\n{\"first_id\":1, \"second_id\":2, \"third_id\":3}"
```

Comment:
I think the correct description would be:
"Receives the dictionary containing the keys to join by and the values to join."
... unless I misunderstood :)